docs/src/main/asciidoc/getting-started-guide.adoc

```diff
@@ -191,6 +207,14 @@ hello
 
 Hit `CTRL+C` to stop the application, but you can also keep it running and enjoy the blazing fast hot-reload.
 
+[TIP]
+Automatically add newline with curl
```

Comment:
This one misses the `.` at the beginning, breaking the TIP layout.

Reply:
fixing.

Reply:
fixed in latest commit.